lstchain/reco/r0_to_dl1.py

```diff
@@ -154,12 +153,28 @@ def get_dl1(
         dl1_container.n_islands = num_islands
         dl1_container.set_telescope_info(subarray, telescope_id)
 
-
-        return dl1_container
-
     else:
-        return None
+        # No image was parametrized, so we put zeros (instead of the default
+        # Nones) in all parameters: a container reset() is not an option because
+        # the default None values prevent the container to be written out. We
+        # cannot use np.nan either, because upon writing it complains for the
+        # integer parameters.
+        #
+        for key in dl1_container.keys():
+            dl1_container[key] = u.Quantity(0, dl1_container.fields[key].unit)
+
+        # Fields width and length do not have in their declaration the units
+        # that are actually expected later in the program, so we set them here.
+        # We now use nans since these are floats, and will be later used to
+        # calculate W/L...
+        dl1_container.width = u.Quantity(np.nan, u.m)
```

Comment:
shouldn't this unit declaration be included in the container definition instead of here?

Reply:
The lstchain DL1 container is deprecated, as you know. The default unit in the definition is degrees, but in the filling (of successfully reconstructed events) meters are used. Hence that line. Of course I could fix all of that from scratch, but it is not the purpose of this PR. The purpose is to keep exactly the same behaviour as before, but assigning the right UCTS info to each event, when possible.